build/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I think we should standardise the ./script folders for supporting executables like that. See http://githubengineering.com/scripts-to-rule-them-all/